content/kubernetes/re-clusters/upgrade-redis-cluster.md

```diff
@@ -78,62 +83,96 @@ customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redis
 deployment.apps/redis-enterprise-operator configured
 ```
 
-### Verify that the operator is running
+### Reapply other manual configurations
+
+When upgrading your cluster, you need to manually reapply the `ValidatingWebhookConfiguration` for the admission controller. See the [Enable the admission controller]({{<relref "/kubernetes/deployment/quick-start#enable-the-admission-controller">}}) step during deployment for more details.
 
-You can verify the operator is running in your namespace by checking the deployment as follows:
 
+If you are using Openshift, you will also need to manually reapply the [Security context constraint](https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html) file ([`scc.yaml`]({{<relref "/kubernetes/deployment/openshift/openshift-cli#deploy-the-operator" >}})).
+
+```sh
+oc apply -f openshift/scc.yaml
 ```
+
+### Verify the operator is running
+
+You can check your deployment to verify the operator is running in your namespace.
+
+```sh
 kubectl get deployment/redis-enterprise-operator
 ```
 
 You should see a result similar to this:
 
-```
+```sh
 NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
 redis-enterprise-operator   1/1     1            1           0m36s
 ```
 
-### 2- Upgrade the Redis Enterprise cluster version
+{{< warning >}}
+ We recommend upgrading the REC as soon as possible after updating the operator. After the operator upgrade completes, the REC is considered frozen. The operator will no longer manage the REC and the REC and its REDB can't be modified until the REC upgrade completes.
+ {{< /warning >}}
 
-Before beginning the upgrade of the Redis Enterprise cluster version, check the K8s operator release notes to find the Redis Enterprise image tag. For example, in Redis Enterprise K8s operator release [6.0.12-5](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.0.12-5), the `Images` section shows the Redis Enterprise tag is `6.0.12-57`.
+ If you have `autoUpgradeRedisEnterprise: true` specified in your REC spec, the operator will automatically trigger the REC upgrade after the operator upgrade completes.
 
-After the Operator upgrade is complete, you can upgrade Redis Enterprise cluster version using the following steps.
+## Upgrade the Redis Enterprise cluster (REC)
 
-1. Run `kubectl edit rec` in the namespace your Redis Enterprise cluster is deployed in.
-2. Replace the `image:` declaration under `redisEnterpriseImageSpec` with the new version tag you found in the operator release notes.
-3. Save the changes to apply
+Before beginning the upgrade of the Redis Enterprise cluster, check the K8s operator release notes to find the Redis Enterprise image tag. For example, in Redis Enterprise K8s operator release [6.0.12-5](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.0.12-5), the `Images` section shows the Redis Enterprise tag is `6.0.12-57`.
```

Comment:
Now that we also mention autoUpgrade, maybe it's better to explicitly distinguish between automatic and manual REC upgrades? At least so that the reader can understand that the below steps are only relevant if using manual upgrade.

Reply:
Essentially there are two methods to upgrade, so we can have two subsections under "Upgrade the REC":
1. Automatic - by setting `autoUpgradeRedisEnterprise: true` (maybe it was already set before).
2. Manually - by retrieving the corresponding RE image from the release notes, and then updating the REC object.